models/workouts-model.js

```diff
@@ -13,13 +13,15 @@ function fetchAllWorkouts(){
 
 function fetchWorkoutByLevel(workoutLevel) {
     return client.connect().then(() => {
-        return workoutsDb.findOne({level: workoutLevel});
+        return workoutsDb.findOne({level: +(workoutLevel)});
```

Comment:
You could also try doing parseInt(workoutLevel) here, or even Number(workoutLevel).